endpoints/get_address_transactions.py

```diff
@@ -103,3 +106,23 @@ async def get_full_transactions_for_address(
         tx_ids_in_page = [x[0] for x in tx_within_limit_offset.all()]
 
     return await search_for_transactions(TxSearch(transactionIds=tx_ids_in_page), fields)
+
+@app.get("/addresses/{kaspaAddress}/transactions-count",
+         response_model=TransactionCount,
+         tags=["Kaspa addresses"])
+async def get_transaction_count_for_address(
+    kaspaAddress: str = Path(
+            description="Kaspa address as string e.g. "
+                        "kaspa:pzhh76qc82wzduvsrd9xh4zde9qhp0xc8rl7qu2mvl2e42uvdqt75zrcgpm00",
+            regex="^kaspa\:[a-z0-9]{61}$")
+):
+    """
+    Count the number of transactions associated with this address
+    """
+
+    async with async_session() as s:
+        count_query = select(func.count(TxAddrMapping.transaction_id)).filter(TxAddrMapping.address == kaspaAddress)
```

Comment:
I tried to `count(*)` and this `count(TxAddrMapping.transaction_id)`. The count(*) has a better timing performance:

```
    async with async_session() as s:
        t1 = time.time()
        count_query = select(func.count(TxAddrMapping.transaction_id)).filter(TxAddrMapping.address == kaspaAddress)
        tx_count = await s.execute(count_query)
        print(time.time() - t1)

        t1 = time.time()
        count_query = select(func.count()).filter(TxAddrMapping.address == kaspaAddress)
        tx_count = await s.execute(count_query)
        print(time.time() - t1)
```

print for kaspa:qpjunp39ssazf4rzfxxu0hd35xggfxn6lq0ls9u9q6peevzcmcv4xmv9q4njd:
11.252254247665405
0.28919363021850586

Reply:
```suggestion
        count_query = select(func.count()).filter(TxAddrMapping.address == kaspaAddress)
```

Reply:
The very tx-heavy addresses need still time. I don't think we can accelerate that.